src/views/Home.vue

```diff
@@ -11,7 +11,7 @@
           :noticeId="noticeId"
           v-bind="notices.data[noticeId]"
         />
-        <button @click="showAllNotices = !showAllNotices" class="home__notices__load-all-btn">
+        <button v-if="showLoadAllBtn" @click="showAllNotices = !showAllNotices" class="home__notices__load-all-btn">
```

Comment:
WHy showAllNotices = !showAllNotices

Reply:
We have showAllNotices = false by default
If we click it makes true and shows all notices
If we want to collapse list - click again and make value false
-> showAllNotices = !showAllNotices